### ISSUE TITLE
Improve colormap in check_audapterLPC

### ISSUE THREAD

**Issue by cwnaber** [carrien/free-speech#128]
According to Carrie, the colormap for the spectrograms in `check_audapterLPC` is ok, but the dynamic range is too low. And at some point, Ben got a rainbow colormap.

The colormap is controlled by `varycolor` on line 293, which we got from the file exchange and is actually in the `current-studies` repo. So that's not great either, that this free-speech function has a current-studies function as a dependency.

>     %Set up colors. Using varycolor allows for any number of vowels
>     plotColors = varycolor(UserData.nVowels);

**Comment by carrien** [comment]
@cwnaber : Good catch on the varycolor dependency; agree that this should be changed to not rely on current-studies. This is unrelated to the spectrogram colormap, though -- it just affects what line/marker colors are used to plot on top of the spectrograms and in the 2D vowel space plot. The colormap is set when the spectrogram is plotted (in this case, by show_spectrogram).

**Comment by cwnaber** [comment]
> This is unrelated to the spectrogram colormap, though -- it just affects what line/marker colors are used to plot on top of the spectrograms and in the 2D vowel space plot. The colormap is set when the spectrogram is plotted (in this case, by show_spectrogram).

In that case, @parrell probably needs to move the "commonmcode" folder to the bottom of his matlab path. There are probably conflicting `show_spectrogram` functions, since we have our own in free-speech\experiment_helpers\show_spectrogram.m. This is something I do in step #7 of [this KB doc](https://kb.wisc.edu/smng/internal/86260) when setting up new computers.
